news/2020-10-29-project-loved-october-2020.md

```diff
@@ -0,0 +1,232 @@
+
+[![](/wiki/shared/news/2020-10-29-project-loved-october-2020/catch/5-starlight-express.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[ginkiha - Starlight Express](https://osu.ppy.sh/beatmapsets/468895#fruits) by [Equim](https://osu.ppy.sh/users/4590473), [Xinely](https://osu.ppy.sh/users/1521445) and [JBHyperion](https://osu.ppy.sh/users/4879508)\
+200 BPM, 1:36 | 2.25★, 3.49★, 3.93★, 6.03★\
+*description written by [Yumeno Himiko](https://osu.ppy.sh/users/1806962)*
+
+This time, my pick is a great map from underrated old mapper Equim. ginkiha is a well-known artist in osu!catch community, and the song *Starlight Express* is also one of his most famous tracks. This map is clean and fun to play, and requires a bit of technical movements for catching the patterns. The map may not be hard as hell by 2020 standards, but it is still an interesting map to enjoy. Welcome aboard *Starlight Express*, and have fun with the fruit journey!
+
+## <a id="mania"></a>osu!mania
+
+osu!mania Loved candidates were chosen by [-mint-](https://osu.ppy.sh/users/8976576), [Abraxos](https://osu.ppy.sh/users/5025064), [lenpai](https://osu.ppy.sh/users/5314573), [Pope Gadget](https://osu.ppy.sh/users/2288341) and [Toaph Daddy](https://osu.ppy.sh/users/7616811)!
+
+---
+
+[![](/wiki/shared/news/2020-10-29-project-loved-october-2020/mania/1-forgotten.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[The Ghost of 3.13 - Forgotten](https://osu.ppy.sh/beatmapsets/338665#mania) by [Shoegazer](https://osu.ppy.sh/users/2520707) and [Zyph](https://osu.ppy.sh/users/1600432)\
+250 BPM, 2:42 | \[4K\], 4.50★ – 5.92★
+
+*În micul orășel, uitat de lume.* Contrary to its name, this chart has hardly been forgotten—the Collab difficulty by Shoegazer and Zyph, mapped in 2015, has garnered quite the popularity, being featured in both Shoegazer's 10th Dan v2 and the infamous Epsilon dan (sped up by 1.4), as well as the finals pool in MWC 2015. It is a very heavy technical map with harsh bursts, rolls, and longjacks, and its ecstatic patterning throughout the entire map keeps players on their toes for the duration of the map. It also has very light stutter SVs which might trip some players up if they aren't careful enough.
+
+The newer difficulty, Extra, is a significantly lighter but no less technical approach in comparison to the Collab difficulty. Mapped in 2020, Extra emphasizes musical accuracy and detail, which it is able to accomplish because of its lighter nature. It features many two-handed trills, rolly bursts, and short anchors. The mapset featuring rates of this difficulty up to 350 BPM (1.4) leans itself to be a more approachable chart for players of a larger skill range, though still being a great challenge for all. It had received a good amount of popularity even prior to its usage in MWC 2020 and continues to be played frequently among competitive players within the mapset's skill range.
```

Comment:
```suggestion
The newer difficulty, Extra, is a significantly lighter but no less technical approach in comparison to the Collab difficulty. Mapped in 2020, Extra emphasizes musical accuracy and detail, which it is able to accomplish because of its lighter nature. It features many two-handed trills, rolly bursts, and short anchors. The mapset featuring rates of this difficulty up to 350 BPM (1.4) leans itself to be a more approachable chart for players of a larger skill range, though still being a great challenge for all. It was already fairly popular even prior to its usage in MWC 2020 and continues to be played frequently among competitive players within the mapset's skill range.
```